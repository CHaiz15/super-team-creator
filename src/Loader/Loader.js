import React from 'react'
import './Loader.scss'

export const Loader = () => {
  return (
    <section>
      <h1 className='creation-title' >Loading...</h1>
    </section>
  )
}