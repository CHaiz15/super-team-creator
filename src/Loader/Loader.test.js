import React from 'react';
import { Loader } from './Loader';
import { shallow } from 'enzyme'

describe('Loader', () => {
  it('should be an instance of the Loader component', () => {
    const wrapper = shallow(<Loader/>);
      expect(wrapper).toMatchSnapshot();
  });
})
