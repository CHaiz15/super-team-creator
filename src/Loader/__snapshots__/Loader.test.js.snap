// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should be an instance of the Loader component 1`] = `ShallowWrapper {}`;
